data.sql

```diff
@@ -112,5 +112,9 @@ INSERT INTO visits (animal_id, vets_id, visit_date)
             ((SELECT id FROM animals WHERE name = 'Blossom'),(SELECT id from vets WHERE name = 'Stephanie Mendez'), '2020-05-24'),
             ((SELECT id FROM animals WHERE name = 'Blossom'),(SELECT id from vets WHERE name = 'William Tatcher'), '2021-01-11');
 
+INSERT INTO visits (animal_id, vets_id, visit_date) SELECT * FROM (SELECT id FROM animals) animal_id, (SELECT id FROM vets) vets_id, generate_series('1980-01-01'::timestamp, '2021-01-01', '4 hours') visit_timestamp;
```

Comment:
* Kindly add an after screenshot for this query `SELECT * FROM visits where vet_id = 2;` as its not included in the PR